docs/index.md

```diff
@@ -17,40 +17,40 @@ Islandora:
 - **Offers flexibility** - As Islandora content is Drupal content, migrations and batch editing can be done through Drupal's built-in migrate framework and controlled vocabularies created using Drupal taxonomies. Contributed Drupal modules such as [Solr Search API](https://www.drupal.org/project/search_api_solr) enable in-site search, and [Matomo Analytics](https://www.drupal.org/project/matomo) provides usage metrics for site analytics.
 - **Is a community** - A [dedicated, active community of users and developers](https://groups.google.com/forum/#!forum/islandora) is working to push new features, collaborate on improvements, design custom solutions, and create extended functionality. Some of these for Islandora 8 take the form of [Recipes](https://github.com/Islandora-Labs/Islandora-Cookbook).
 
-## Demo
+## Try Islandora
 
-Visit [http://future.islandora.ca/](http://future.islandora.ca/).
+### Sandbox
 
-## Quickstart
+To try Islandora without installing anything, visit [http://future.islandora.ca/](http://future.islandora.ca/). Anyone can log in to this sandbox as an administrator (credentials are below and on the front page) and explore the interface! However, this site is refreshed periodically so your changes will not be permanent. This site uses Islandora Defaults, a way of setting up Islandora for demonstration purposes - this is not the only way that Islandora can be made to work!
 
-These instructions use Vagrant and the [Islandora Ansible playbook](https://github.com/Islandora-Devops/islandora-playbook) to spin up a local development machine, which includes the [Islandora Default site configuration](https://github.com/Islandora/islandora_defaults). See the [Automatic Provisioning](installation/playbook/) page for full instructions.
-[//]: # (Move this link to contributing/hacking-on-islandora.md once that's de-CLAWed.)
+- username: Test
+- password: islandora
 
+### Virtual Machine Image
 
-### Requirements
-- [Virtual Box](https://www.virtualbox.org/)
-- [Vagrant](https://www.vagrantup.com/) (version 2.0 or higher required)
-- [Git](https://git-scm.com/)
-- [OpenSSL](https://www.openssl.org/)
-- [Ansible](https://www.ansible.com/community) (up to, and not past, 2.8.7)
+To try Islandora locally, you can download and run the latest [community sandbox VM image](https://islandora.ca/try). This requires installing VirtualBox or another virtualization provider. This also uses Islandora Defaults, as it is a pre-built version of the Islandora Playbook.
```

Comment:
Add "along with sample content, Views, Blocks, and Drupal contributed modules and themes, to enhance its role as a sandbox."